watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -552,6 +558,14 @@ def scale_variables(m):
     scale_variables(m)
     iscale.calculate_scaling_factors(m)
 
+    # Touch treated properties
+    m.fs.Treated.properties[0].TSS
+    m.fs.Treated.properties[0].COD
+    m.fs.Treated.properties[0].BOD5
+    m.fs.Treated.properties[0].SNKj
+    m.fs.Treated.properties[0].SP_organic
+    m.fs.Treated.properties[0].SP_inorganic
```

Comment:
You should touch these before the solve.

Reply:
My mistake- I misread the GH diff and it looked as if these lines were after the solve. That said, I wonder if it makes more sense to touch before applying scaling. Right now, I think you made each an expression so it might not matter. If they were reverted back to vars, then it would matter.